internal/network/command_network_link_service_create.go

```diff
@@ -32,12 +36,16 @@ func (c *command) newNetworkLinkServiceCreateCommand() *cobra.Command {
 	pcmd.AddOutputFlag(cmd)
 
 	cobra.CheckErr(cmd.MarkFlagRequired("network"))
+	cmd.MarkFlagsOneRequired("accepted-networks", "accepted-environments")
```

Comment:
Both examples use `--accepted-environments`. Can we change one of them to use `--accepted-networks`?